### PR TITLE
[Hotfix] Check user is in CENNZnet Discord server

### PR DESCRIPTION
**Pull request by aidan-starke**
## Summary

 - Handle edge case where user is not a member of CENNZnet discord server

### CODE REVIEW

File: libs/web/components/src/IdentityFormDialog.tsx [hunk 1]
```diff
@@ -29,8 +29,11 @@ export const IdentityFormDialog: FC<IdentityFormDialogProps> = ({
 				<Choose>
 					<Choose.When condition={formState?.status === "Ok"}>
 						<p className="prose text-center">
-							Your identity has been successfully set. Visit Discord to view the
-							Governance channels with your new role!
+							Your identity has been successfully set.&nbsp;
+							<If condition={!!formState?.cennzDiscordMember}>
+								Visit Discord to view the Governance channels with your new
+								role!
+							</If>
```
Comment (ken-futureverse) [inline libs/web/components/src/IdentityFormDialog.tsx:36]: Should keep the message but just omit the `"with your new role!"`

File: apps/gov-website/pages/api/identity/judgement.tsx [hunk 2]
```diff
@@ -71,6 +71,9 @@ export default withMethodGuard(
 
 			return res.json({ ok: true });
 		} catch (error) {
+			if (error?.message === "DISCORD_USER_NOT_FOUND")
+				return res.json({ ok: true, message: error.message });
```
Comment (ken-futureverse) [inline apps/gov-website/pages/api/identity/judgement.tsx:75]: OK approach but not ideal, this should come back as an error, and let the UI catch and handle that issue. will need to update this when doing #40

**Review by ken-futureverse — CHANGES_REQUESTED** [review]

File: libs/web/components/src/IdentityFormDialog.tsx (hunk 1 above)
Comment (aidan-starke) [reply libs/web/components/src/IdentityFormDialog.tsx:36]: I believe they won't have access to the channels without the role

Comment (ken-futureverse) [reply libs/web/components/src/IdentityFormDialog.tsx:36]: Okay, then can tweak the message and send user to the generic channel instead? Feels like we remove the message for this scenario but will offer a CTA to go to Discord

**Pull request closed without merging**